src/cloud-api-adaptor/pkg/securecomms/ppssh/ppssh.go

```diff
@@ -104,21 +104,36 @@ func attestationPhase(listener net.Listener, inbounds sshproxy.Inbounds, outboun
 	//peer.Wait()
 }
 
+// InitSshServer initializes an SSH Server at the PP
+// inbound_strings is a slice of strings where each string is an inbound tag
+// outbounds_strings is a slice of strings where each string is an outbound tag
+// Structure of an inbound tag: "<MyPort>:<InboundName>:<Phase>"
+// Structure of an outbound tag: "<DesPort>:<DesHost>:<outboundName>:<Phase>"
+// Phasde may be "A" (Attestation), "K" (Kubernetes), or "B" (Both)
```

Comment:
Typo: "Phase" (vs. "Phasde")

Phase is apparently used in user-visible errors, so maybe spell it out?